Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -225,7 +227,7 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
                     using (tracer.Step("LibGit2SharpRepository Fetch"))
                     {
                         // This will only retrieve the "master"
-                        repo.Network.Fetch(remote);
+                        LibGit2Sharp.Commands.Fetch(repo, remote.Name, remote.RefSpecs.Select(r => r.Specification), new FetchOptions(), "LibGit2SharpRepository Fetch");
```

Comment:
I haven't heard back on this. I'm thinking we could just run all the tests we got and if they all pass, merge it?